src/MeetingsSDKAdapter.js

```diff
@@ -194,6 +211,116 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     });
   }
 
+  /**
+   * Attempts to mute the microphone of the given meeting ID.
+   * If the microphone is successfully muted, an audio mute event is dispatched.
+   *
+   * @param {string} ID ID of the meeting to mute audio
+   * @memberof MeetingsSDKAdapter
+   */
+  muteAudioMeeting(ID) {
+    const sdkMeeting = this.fetchMeeting(ID);
+
+    sdkMeeting
+      .muteAudio()
+      .then(() => {
```

Comment:
Can we use `async/await` syntax? I don't like that we're mixing styles

Reply:
will address in my upcoming PR